src/runtime/storage/tests/reference-mode-store-test.ts

```diff
@@ -172,9 +187,7 @@ describe('Reference Mode Store', async () => {
     const reference: Reference = {storageKey: new MockHierarchicalStorageKey(''), id: 'an-id', version: {[actor]: 1}};
     referenceCollection.applyOperation({type: CollectionOpTypes.Add, clock: {me: 1}, actor: 'me', added: reference});
 
-    const entityCRDT = new MyEntityModel();
-    entityCRDT.applyOperation({type: EntityOpTypes.Set, field: 'age', value: {id: '42', value: 42}, actor, clock: {[actor]: 1}});
-    entityCRDT.applyOperation({type: EntityOpTypes.Set, field: 'name', value: {id: 'bob'}, actor, clock: {[actor]: 1}});
+    const entityCRDT = myEntityToMyEntityModel(entity, actor);
```

Comment:
you may not need this any more, now that fast-forwarding is built into the CRDT

Reply:
Fast forwarding at the moment only updates the version number for all fields each time an operation is applied. When the fastForward operation can apply multiple operations at the same time, I'll replace the `meEntityToMyEntityModel` function with a fastForward operation. I plan to make this change in a follow up PR.

Reply:
got it, thanks!